virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -72,6 +72,23 @@ The SVVP Certification applies to:
 
 //CNV-28720 Release note: CHANGE
 
+//CNV-28724 Release note: storage profile defaults
+* `claimPropertySets` are now populated by default in storage profiles for the following provisioners:
```

Comment:
We should say that these were the provisioners added.  We already have NetApp Trident, ODF, etc.  I just don't want to make it look like the only ones we support are those that are listed here.

Reply:
Makes sense. I'll insert the word "additional" before "provisioners." Thanks.